python/paddle/fluid/dygraph/layers.py

```diff
@@ -1028,6 +1033,35 @@ def forward(self, input):
             self._parameters[name] = parameter
         return parameter
 
+    def _set_op_attrs(self, attrs):
+        """
+        Add customized attribute while append_op. In case of quantization, we want to save
+        some attributes into op_desc while exporting inference model by @to_static.
+
+        Arguments:
+            attrs(dict): customized attributes that will be added into op_descs.
+
+        NOTE: The interface is only exposed to developers.
+        """
+        if not isinstance(attrs, dict):
+            raise TypeError("attrs should be type(dict), but received {}".
+                            format(type(attrs).__name__))
+
+        if len(self._customized_attrs) != 0:
+            raise RuntimeError("Unsupported to call self._set_op_attrs twice!")
```

Comment:
Can this restriction be removed？ The new attributes can overwrite old attributes. 
Maybe it should analyze whether the hooks have been registered

Reply:
Removed it.